README.md

```diff
@@ -123,16 +123,16 @@ Check [the specification][AluVM] for the details.
 
 ## History
 
-- The need for AluVM recognized as a part of RGB project on the
+- The need for AluVM recognized as a part of RGB project in
   Mar, the 24 & 31st, 2021 (see developers call <https://youtu.be/JmKNyOMv68I>)
 - Concept was presented on 19th of May 2021 
   ([check the recoding](https://youtu.be/Mma0oyiVbSE))
 - v0.1 release of Rust AluVM implementation on the 28th of May 2021
   ([ISA & API docs](https://docs.rs/aluvm/0.1.0/alure/))
 - v0.2 release with multiple enhancements on the 9 Jun 2021
   ([ISA & API docs](https://docs.rs/aluvm/0.2.1/aluvm/)) – see presentation
-  on [Youtube] or read [slides]
+  on [Youtube] or read [slides](https://github.com/LNP-BP/presentations/blob/master/Presentation%20slides/RGB%20VM%20%26%20scripting.pdf)
```

Comment:
Sorry, we are still duplicating this link URL, which will result in a fact that if someone will change the link at the end of the document, this will remain unchanged

Reply:
Deleted the link. My bad!